test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -14,148 +14,172 @@ tags:
     ```
 
-    Replace `numRequests` and `interval` for the values from the following table:
-
-    > These figures are obtained to make the alert fire for ~2 minutes, by calculating
-    > the interval between requests to keep a rate of n request/minute where n
-    > is in the range for the alert
-    >
-    > _Example_: To trigger level 1 we use a rate of 85 (between 80% and 90% of the limit),
-    > so we perform 170 requests every 0.71s. That is:
-    >
-    > - `numRequests = expected rate x minutes to keep the alert firing`
-    > - `interval = 60 / expected rate`
-
-    | Alert to fire                        | `numRequests` | `interval` |
-    | ------------------------------------ | ------------- | ---------- |
-    | RHOAMApiUsageLevel1ThresholdExceeded | 170           | 0.71       |
-    | RHOAMApiUsageLevel2ThresholdExceeded | 186           | 0.645      |
-    | RHOAMApiUsageLevel3ThresholdExceeded | 240           | 0.5        |
-
-    > Only the requests for level 3 should be eventually failing with a `429 Too Many Requests` status code
-    > due to the rate limit being exceeded
-
-13) Run the following command and log in to Prometheus service (as a kubeadmin)
-
-```sh
-open "https://$(oc get routes prometheus-route -n redhat-rhmi-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
-```
-
-14. Go to Alerts -> and search for corresponding the alert (Alert to fire in the table)
-15. Click on the alert name -> click on the expression (link) -> Graph
-16. Run the corresponding script to trigger the alert
-17. Verify that after a minute the alert graph is showing some data
-    > In order to verify the current rate, run the following query in the Graph tab
-    >
-    > ```
-    > increase(ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits[1m]))
-    > ```
-    >
-    > The alert should fire when the value is in the expected range
-18. Repeat the same process for the remaining alerts and verify that they are firing as expected. Make sure to wait at least a minute before running each script.
+    Select the Rate Limiting dashboard
+
+    Run the following command:
+
+    ```sh
+    for i in {1..150}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>; done
+    ```
+
+    _NOTE:_ The above command should eventually result in failing `429 Too Many Requests` status codes. This is to be expected. If no requests have been rejected make sure to check the current Rate Limit configuration in the `sku-limits-managed-api-service` configmap of the `redhat-rhoam-operator` namespace
+
+    While the command is running, check that the graphs in the Grafana dashboard are updating every minute. For example, the "Last 1 Minute - Rejected" percentage should be above 90%
+
+    See example:
+
+    ![](https://user-images.githubusercontent.com/4881144/99069295-5e34fc00-25ae-11eb-874b-3cf5c4a03736.png)
```

Comment:
```suggestion
    See [this example](https://user-images.githubusercontent.com/4881144/99288530-07dced00-283c-11eb-9cba-906151dd7dfb.png)
```
^ I would add a web link to the file, since the test cases will accessed from JIRA